guides/plugins/apps/app-base-guide.md

```diff
@@ -331,6 +331,33 @@ You can verify the authenticity of the incoming request by checking the `shopwar
 
 You can use a variety of events to react to changes in Shopware that way. See that table [Webhook-Events-Reference](../../../resources/references/app-reference/webhook-events-reference.md) for an overview.
 
+### App Notification
+Starting from Shopware version 6.4.6.0, If you want to send notifications to the admin, to inform the user about some actions that happened on the app side, the app should send a POST request to the `api/notification` endpoint with a valid body and the header `Authorization` token.
+Your app can request 10 times before being delayed by the system.
+
+After 10 attempts you need to wait 10 seconds before trying to make requests again.
+After 15 attempts it's 30 seconds.
+After 20 attempts it's 60 seconds.
+After 24 hours without a failed request the limit is reset.
+
+Examples request body:
+You need to pass the `status` property, the content of the notification as `message` property and you should pass the email of the admin that will be sent notifications as `adminEmail` inside the payload.
+If you don't send the `adminEmail`, the notifications will be displayed to every admin.
```

Comment:
```suggestion
If you don't send the `adminEmail`, the notifications will be displayed to the first user that logs in.
```
Or will it be shown to multiple users?

Reply:
Yes. It will be shown to every users because we save the latest reading time to user_config table for each user every time they read. So we can fetch new notifications for each user when they login

Reply:
Then rephrase that to `every user.`
as admin can be misleading as meaning user with admin permissions